docs/reference/commands.md

```diff
@@ -0,0 +1,104 @@
+---
+id: commands
+title: Command reference
+description: A Command is a requested action issued by a Worker to the Temporal Cluster, after the Worker has finished completing the execution of a Workflow Task.
```

Comment:
"finished completing the execution" is hard to parse for me.

Reply:
reworded